src/appleseed/renderer/kernel/texturing/texturestore.cpp

```diff
@@ -77,6 +78,19 @@ StatisticsVector TextureStore::get_statistics() const
     return StatisticsVector::make("texture store statistics", stats);
 }
 
+Dictionary TextureStore::get_params_metadata()
+{
+    Dictionary metadata;
+    metadata.dictionaries().insert(
+        "max_size",
```

Comment:
You're right, let's keep as is for now.

Maybe in the future it'll make sense to expose MB instead of bytes everywhere where it makes sense (such as for texture caches), but for now it's fine, and we're consistent.